ui/dashboards/src/components/Panel/Panel.tsx

```diff
@@ -66,6 +81,7 @@ export function Panel(props: PanelProps) {
             sx={{
               display: 'flex',
               alignItems: 'center',
+              height: '20px',
```

Comment:
Nit: This feels a little like a "magic number" to me and like we don't want to hardcode it, but maybe I'm wrong? If it's hardcoded, could be difficult for folks with a different theme to make it look correct (e.g. different typography, etc.)

Reply:
I updated it to using `minHeight`. This is because there is a bug right now where the panel header height is different depending if there is the info icon.